mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -468,58 +469,109 @@ class LowerCastRewrite : public OpRewritePattern<graphblas::CastOp> {
   };
 };
 
-class LowerTransposeRewrite : public OpRewritePattern<graphblas::TransposeOp> {
+class LowerTransposeLayoutDWIMRewrite
```

Comment:
I should probably rename the `Lower[OP_NAME]LayoutDWIMRewrite` classes to `[OP_NAME]LayoutDWIMRewrite` instead.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/d5e602fea2f3e2514222e7b65c8dbda36922fbd6 implements this suggestion.